tests/test_remote.py

```diff
@@ -42,12 +43,19 @@ def setUpModule():   # pylint: disable=invalid-name
     bootstrap.setup_component(
         hass, http.DOMAIN,
         {http.DOMAIN: {http.CONF_API_PASSWORD: API_PASSWORD,
-         http.CONF_SERVER_PORT: MASTER_PORT}})
+                       http.CONF_SERVER_PORT: MASTER_PORT}})
 
     bootstrap.setup_component(hass, 'api')
 
-    hass.start()
-    time.sleep(0.05)
+    def block_start(hass_obj):
```

Comment:
Interesting, maybe releasing the GIL with such a sleep() fix some of my busy threads issues:

`WorkerPool:All 12 threads are busy and 73 jobs pending`

It always seems to happen around start  & adding an array of many devices `add_devices` (and their state updates)